src/script.js

```diff
@@ -60,25 +48,8 @@ $j(document).ready(() => {
 	window.addEventListener('focus', G.onFocus.bind(G), false);
 
 	// Add listener for Fullscreen API
-	$j('#fullscreen').on('click', () => {
-		if (isNativeFullscreenAPIUse()) {
-			disableFullscreenLayout();
-			document.exitFullscreen();
-		} else if (!isNativeFullscreenAPIUse() && window.innerHeight === screen.height) {
-			alert('Use F11 to exit fullscreen');
-		} else {
-			enableFullscreenLayout();
-			$j('#AncientBeast')[0].requestFullscreen();
-		}
-	});
-
-	window.addEventListener('resize', () => {
-		if (window.innerHeight === screen.height && !$j('#fullscreen').hasClass('fullscreenMode')) {
```

Comment:
Since `F11` key does not trigger the `fullscreenchange` event, I don't think we should "support" this feature by using hacky code to detect it.

This code did not work on my browsers (Debian Linux box, Chromium, Chrome and Firefox).

We should use the Fullscreen API and wait for the browser vendors to fix it properly. Unfortunately this decision will cause small UI issues as you described, @DreadKnight.

What do you think about it?